### ISSUE TITLE
fix: Additional role=combobox in Combobox web component

### ISSUE THREAD

**Issue by jumarroq** [microsoft/fast#4824]
# 🐛 Bug Report

Two role="combobox" are added to the Combobox web component, one in the shadowroot on the input element, and another on the template/host. The one on the template/host I don't know how it gets added, the one on the input element it's obvious from the code and it has the accompanying aria-expanded attribute as expected.

I would expect only one to be there and the missing aria-expanded attribute seems to be an a11y violation.

## 💻 Repro or Code Sample

https://explore.fast.design/components/fast-combobox

## 🤔 Expected Behavior

Only one role="combobox" should be rendered per webcomponent

## 😯 Current Behavior

two are being rendered:

![image](https://user-images.githubusercontent.com/68871578/120732923-05ba4300-c49b-11eb-9750-b55e233accae.png)

## 💁 Possible Solution

From experimenting, it looks like the Combobox is inheritting from ListBox which defines the role class variable as @attr, not sure if there's a way to undo that, but that is causing the role to be rendered in the dom on the host and then the combobox uses that to also render the input's role. We would either need to undo that attr directive or not inherit from ListBox

## 🔦 Context

Getting accessibility violations in the a11y insights tool. Looks like NVDA is also not reading the expanded attribute correctly either.

## 🌍 Your Environment

* OS & Device: Windows
* Browser Edge and Chromium
